src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$card-width: 200px;
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  gap: 48px;
+  justify-content: center;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
+    display: grid;
+    place-items: center;
+    width: $card-width;
```

Comment:
width is redundant here

Reply:
it's necessary for viewport < 488px

Reply:
Ok. Now I see it. Sorry)

Reply:
if you add `grid-template-columns: repeat(1, $card-width);` to .catalog than you can remove width